### PR TITLE
chore: generate new client

### PR DESCRIPTION
**Pull request by JoeWang1127**

### CODE REVIEW

**Comment by JoeWang1127** [comment]
@suztomo, this is the generated library after the [fix](https://github.com/googleapis/synthtool/pull/1770) of post-processor.

Java GraalVM Native Image test failed with the following output, it seems like not relevant to the generated library?

```
[INFO] Reactor Summary:
[INFO]
[INFO] Google Cloud Vertex AI Parent 3.13.0-SNAPSHOT ...... SUCCESS [  1.458 s]
[INFO] proto-google-cloud-aiplatform-v1beta1 0.29.0-SNAPSHOT SUCCESS [  1.161 s]
[INFO] proto-google-cloud-aiplatform-v1 3.13.0-SNAPSHOT ... SUCCESS [  0.135 s]
[INFO] grpc-google-cloud-aiplatform-v1beta1 0.29.0-SNAPSHOT SUCCESS [  0.086 s]
[INFO] grpc-google-cloud-aiplatform-v1 3.13.0-SNAPSHOT .... SUCCESS [  0.087 s]
[INFO] Google Cloud Vertex AI 3.13.0-SNAPSHOT ............. SUCCESS [07:52 min]
[INFO] Google Compute Engine Parent 1.22.0-SNAPSHOT ....... SUCCESS [  0.053 s]
[INFO] proto-google-cloud-compute-v1 1.22.0-SNAPSHOT ...... SUCCESS [  0.228 s]
[INFO] Google Compute Engine 1.22.0-SNAPSHOT .............. FAILURE [23:24 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  31:25 min
[INFO] Finished at: 2023-02-24T01:30:09Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.graalvm.buildtools:native-maven-plugin:0.9.18:test (test-native) on project google-cloud-compute: Execution of /tmpfs/src/github/google-cloud-java/java-compute/google-cloud-compute/target/native-tests --xml-output-dir /tmpfs/src/github/google-cloud-java/java-compute/google-cloud-compute/target/native-test-reports -Djunit.platform.listeners.uid.tracking.output.dir=/tmpfs/src/github/google-cloud-java/java-compute/google-cloud-compute/target/test-ids returned non-zero result -> [Help 1]
```

**Comment by suztomo** [comment]
Can you retry?

File: java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.maps</groupId>
```
Comment (JoeWang1127) [inline java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml:4]: checked

File: java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  </description>
+
+  <properties>
+    <maven.antrun.skip>true</maven.antrun.skip>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.maps</groupId>
```
Comment (JoeWang1127) [inline java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml:28]: checked

File: java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <artifactId>google-maps-mapsplatformdatasets</artifactId>
+        <version>0.0.1-SNAPSHOT</version><!-- {x-version-update:google-maps-mapsplatformdatasets:current} -->
+      </dependency>
+      <dependency>
+        <groupId>com.google.maps.api.grpc</groupId>
```
Comment (JoeWang1127) [inline java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml:33]: checked

File: java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <artifactId>grpc-google-maps-mapsplatformdatasets-v1alpha</artifactId>
+        <version>0.0.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-maps-mapsplatformdatasets-v1alpha:current} -->
+      </dependency>
+      <dependency>
+        <groupId>com.google.maps.api.grpc</groupId>
```
Comment (JoeWang1127) [inline java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml:38]: checked

File: java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml [hunk 5]
```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.maps</groupId>
```
Comment (JoeWang1127) [inline java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml:4]: checked

File: java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+  <artifactId>google-maps-mapsplatformdatasets</artifactId>
+  <version>0.0.1-SNAPSHOT</version><!-- {x-version-update:google-maps-mapsplatformdatasets:current} -->
+  <packaging>jar</packaging>
+  <name>Google Maps Platform Datasets API</name>
+  <description>Maps Platform Datasets API Maps Platform Datasets API.</description>
+  <parent>
+    <groupId>com.google.maps</groupId>
```
Comment (JoeWang1127) [inline java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml:11]: checked

File: java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-common-protos</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.maps.api.grpc</groupId>
```
Comment (JoeWang1127) [inline java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml:45]: checked

File: java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+
+    <!-- Test dependencies -->
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.maps.api.grpc</groupId>
```
Comment (JoeWang1127) [inline java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml:89]: checked

**Review by JoeWang1127 — COMMENTED** [review]
I checked the following group ids.

**Comment by JoeWang1127** [comment]
The GraalVM Native Image test passed the 2nd time.

**Comment by suztomo** [comment]
I merged main to see group_id_check_for_maps_libraries result.

**Comment by suztomo** [comment]
It passed:

https://github.com/googleapis/google-cloud-java/commit/27248c8659bea478bb5c0527812742f98916e316

```
Passed java-maps-addressvalidation/pom.xml
Passed java-maps-addressvalidation/grpc-google-maps-addressvalidation-v1/pom.xml
Passed java-maps-addressvalidation/google-maps-addressvalidation/pom.xml
Passed java-maps-addressvalidation/proto-google-maps-addressvalidation-v1/pom.xml
Passed java-maps-addressvalidation/google-maps-addressvalidation-bom/pom.xml
Passed java-maps-mapsplatformdatasets/pom.xml
Passed java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets-bom/pom.xml
Passed java-maps-mapsplatformdatasets/google-maps-mapsplatformdatasets/pom.xml
Passed java-maps-mapsplatformdatasets/proto-google-maps-mapsplatformdatasets-v1alpha/pom.xml
Passed java-maps-mapsplatformdatasets/grpc-google-maps-mapsplatformdatasets-v1alpha/pom.xml
Passed java-maps-routing/pom.xml
Passed java-maps-routing/google-maps-routing/pom.xml
Passed java-maps-routing/proto-google-maps-routing-v2/pom.xml
Passed java-maps-routing/google-maps-routing-bom/pom.xml
Passed java-maps-routing/grpc-google-maps-routing-v2/pom.xml
All group IDs start with com.google.maps. Good.
```

**Pull request closed without merging**